app/keymaps/base.json

```diff
@@ -57,5 +57,7 @@
   "contenteditable:outdent": "mod+[",
   "contenteditable:indent": "mod+]",
   "contenteditable:next-selection": "mod+\"",
-  "contenteditable:open-spelling-suggestions": "mod+m"
+  "contenteditable:open-spelling-suggestions": "mod+m",
+
```

Comment:
no need for this blank line. Keep it all together in a single block for `contenteditable`

Reply:
Again this is for consistency (all core, contenteditable seemed to be blocked together. But should this `"composer:select-attachment": "mod+shift+a"` be in https://github.com/Foundry376/Mailspring/blob/84643cb3c4b1dd281ec005cddc95ad302266ef31/app/internal_packages/composer/keymaps/composer.json ?

Reply:
heh you are right again. When I read this I read it as part of the contenteditable block. Derping it up!